tests/acceptance/editor.feature

```diff
@@ -22,3 +22,12 @@ Background:
     And I publish the post
     Then I see the video in the editor
     And I see the video in the post published
+
+  Scenario: Import a campaign
+    Given I am on a import campaign page
+    When I select xml file to import "test-campaign.lorem-ipsum.xml"
+    And I upload file and import
+    And I checked checkbox "#import-attachments"
```

Comment:
Here I wouldn't put anything so specific as an id, you can write "And I select the import of attachments". Anything that a user cannot see should probably stay hidden in the implementation, the user would check a checkbox with label _X_, or the _first_ checkbox, etc. But we don't even have to talk about checkbox, we can say "And I select these options", so that if we change this form to different inputs the scenario stays valid (we'll still have to change our implementation, but the feature itself won't move)